networkx/algorithms/coloring/greedy_coloring.py

```diff
@@ -131,7 +127,7 @@ def strategy_connected_sequential(G, colors, traversal='bfs'):
     strategy_connected_sequential_dfs method). The default is bfs.
     """
     for component_graph in nx.connected_component_subgraphs(G):
-        source = component_graph.nodes()[0]
+        source = list(component_graph)[0]
```

Comment:
@dschult You're right, I will go through all the changes again to spot these mistakes :)